src/content/reference/cluster-upgrades/_index.md

```diff
@@ -64,14 +64,47 @@ Azure, AWS, and KVM based installations have independent versioning systems, as
 
 ### Determining the release version and inspecting release details
 
-As a user with access to the Giant Swarm API and being member of the organization owning the cluster, you can use both the web UI or the CLI to find out what release version your tenant cluster currently has.
+As a user with access to the Giant Swarm API and being a member of the organization owning the cluster, you can use both the web UI or the CLI to find out what release version your tenant cluster currently has.
 
 In the web UI, both the cluster overview and the cluster details page show the release version number. 
 In the cluster details page you can click the release version number to get more information about a release.
 
 Likewise in the CLI, commands like [`gsctl list clusters`](/reference/gsctl/list-clusters/) and [`gsctl show cluster`](/reference/gsctl/show-cluster/) reveal the release version number.
 To get information on all available releases, use the [`gsctl list releases`](/reference/gsctl/list-releases/) command.
 
+### Upgrage a cluster
+
+As an authenticated user you can upgrade the cluster version using the web UI or the CLI. The web UI shows a yellow link next to the version information once there is an upgrade available. For the CLI you can use the command [`gsctl upgrade cluster`](/reference/gsctl/upgrade-cluster/) for the same purpose.
```

Comment:
```suggestion
As an authenticated user you can upgrade the cluster version using the web UI or the CLI. The web UI shows a yellow link next to the version information if there is an upgrade available. For the CLI you can use the command [`gsctl upgrade cluster`](/reference/gsctl/upgrade-cluster/) for the same purpose.
```

Not sure but should we show the command output here? I'm assuming `gsctl upgrade cluster` shows if an upgrade is available and then you confirm. But users might find this command scary at first.

I should also stop using curl for this 😅